### PR TITLE
Render href in text thumbnails (test results)

### PR DESCRIPTION
**Pull request by Martchus**
Fix https://progress.opensuse.org/issues/14544

### CODE REVIEW

File: templates/step/viewtext.html.ep [hunk 1]
```diff
@@ -1,3 +1,3 @@
 % use Mojo::Util qw(xml_escape);
 % use OpenQA::Utils;
-<pre><%== bugref_to_href(xml_escape($textresult)) %></pre>
+<pre><%== bugref_to_href(url_to_href(xml_escape($textresult))) %></pre>
```
Comment (coolo) [inline templates/step/viewtext.html.ep]: that's an interesting cascade :)

File: lib/OpenQA/Utils.pm [hunk 2]
```diff
@@ -417,6 +419,12 @@ sub href_to_bugref {
     return $text;
 }
 
+sub url_to_href {
+    my ($text) = @_;
+    $text =~ s(($RE{URI}))(<a href="$1">$1</a>)gx;
+    return $text;
+}
```
Comment (okurz) [inline lib/OpenQA/Utils.pm]: Don't you want to share code with lib/OpenQA/Schema/Result/Comments.pm without duplicating?

Comment (Martchus) [inline lib/OpenQA/Utils.pm]: Ok, I'll do this later.

Comment (Martchus) [inline lib/OpenQA/Utils.pm]: Seems to be two different things. Here I want to replace *any* URL with corresponding `a` tag. In `Comments.pm` only URLs which aren't already in HTML tags or Markdown are replaced with Markdown syntax.

**Pull request merged**